src/snowflake/snowpark/session.py

```diff
@@ -434,12 +434,12 @@ def conf(self) -> RuntimeConfig:
 
     @property
     def sql_simplifier_enabled(self) -> bool:
+        """Set to ``True`` to use the SQL simplifier.
+        The generated SQLs from ``DataFrame`` transformations would have fewer layers of nested queries if the SQL simplifier is enabled."""
```

Comment:
Only the getter function is rendered in documentation.

https://docs.snowflake.com/developer-guide/snowpark/reference/python/latest/api/snowflake.snowpark.Session.sql_simplifier_enabled

Reply:
Should we also mention that by default this is true? Note that this is not technically correct because it is controlled by a server param, but I don't see we are going to disable it any more. cc @sfc-gh-yixie

Reply:
This makes sense to me, I have added that it defaults to true.